dotcom-rendering/src/palette.ts

```diff
@@ -3763,6 +3763,9 @@ const matchStatsBackground: PaletteFunction = ({ design }) => {
 	}
 };
 
+const matchStatsBackgroundDark: PaletteFunction = () =>
+	sourcePalette.sport[300];
```

Comment:
Feels a bit bright to my eye 👁️

Reply:
This does look on the bright side for a dark mode. As these pages are not rendered by DCAR yet, let's revisit when we are ready for this to go live.